src/compiler/moduleNameResolver.ts

```diff
@@ -29,6 +29,7 @@ namespace ts {
         path: string;
         extension: Extension;
         packageId: PackageId | undefined;
+        originalPath?: string | true;

```

Comment:
Could you add a doc comment explaining the meaning of the three possible types this has -- `true`, `string`, and `undefined`?